cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/BasicDatastore.java

```diff
@@ -643,7 +643,7 @@ public LocalDocument call(SQLDatabase db) throws Exception {
         return null;
     }
 
-    private BasicDocumentRevision createDocument(SQLDatabase db,String docId, final DocumentBody body)
+    private BasicDocumentRevision createJsonDocument(SQLDatabase db, String docId, final DocumentBody body)
```

Comment:
What's the thinking behind the renaming, to make clear they are operating on the serialised json? Given that we use the term 'JSON' fairly loosely in methods and member names I wonder whether a suffix like 'Internal' or 'InDatabase' or similar might be more apt